config/examples/currencies.yml

```diff
@@ -0,0 +1,13 @@
+currencies: &default
+ USD - American Dollar: 'USD'
+ EUR - Euro: 'EUR'
```

Comment:
For this:
https://github.com/3scale/porta/blob/46ac8701c641320d608777761f7bb9c793bfd640/test/factories/billing_strategy.rb#L3

Not sure if I should keep this example or remove it and change the factory to use USD.